.gitignore

```diff
@@ -11,7 +11,10 @@ __pycache__/
 .Python
 env/
 build/
+build_debug/
```

Comment:
Please don't change this. You can change it locally, but don't upsteam.

Reply:
ok, I'll fix this.